ts/src/pro/test/Exchange/test.watchOHLCV.ts

```diff
@@ -21,7 +21,7 @@ async function testWatchOHLCV (exchange, skippedProperties, symbol) {
         let response = undefined;
         try {
             response = await exchange.watchOHLCV (symbol, chosenTimeframeKey, since, limit);
-        } catch (e) {
+        } catch (e: any) {
```

Comment:
why do we need this cast?

Reply:
Apparently we don't, but I feel like I was receiving an error before without the cast

Reply:
I think this is why https://ci.appveyor.com/project/ccxt/ccxt/builds/49701606